apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerCommand.cs

```diff
@@ -172,6 +172,34 @@ public object Clone()
         /// <inheritdoc />
         public override int ExecuteNonQuery() => ExecuteNonQueryAsync(_synchronousCancellationTokenSource.Token).ResultWithUnwrappedExceptions();
 
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <returns></returns>
+        public Task<DbDataReader> ExecuteReaderAsync(TimestampBound singleUseReadSettings)
+        {
+            return ExecuteReaderAsync(singleUseReadSettings, CancellationToken.None);
+        }
+
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <returns></returns>
+        public async Task<DbDataReader> ExecuteReaderAsync(TimestampBound singleUseReadSettings, 
+            CancellationToken cancellationToken)
```

Comment:
I'm not sure I follow. The base class has 4 other `ExecuteReaderAsync` overloads, and yes they are in pairs: one with the `CancellationToken` and one without, but I don't see how it affects these overloads. Unless these are meant to shadow two of the overloads on the base, in which case they should have the `new` modifier, but that doesn't seem to be the case.

That said, I don't feel too strongly about it, so if you think it's best to keep them separated, then by all means.